<!DOCTYPE html>
<html lang="en"><head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1"><!-- Begin Jekyll SEO tag v2.5.0 -->
<title>About GET Labs | GET :: Genomes Environments Traits</title>
<meta name="generator" content="Jekyll v3.8.5" />
<meta property="og:title" content="About GET Labs" />
<meta name="author" content="GitHub User" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="For the past six years, this event has explored the frontiers of understanding about human biology. It serves as an annual forum to debate the technical, commercial, and societal impacts of advances in our ability to measure and understand people and their traits. The GET Conference is produced by PersonalGenomes.org, a 501(c)(3) charitable organization based in the United States whose mission is to make a wide spectrum of human biological information accessible and actionable to increase biological literacy and improve human health. We support groundbreaking initiatives including the Global Network of Personal Genome Projects, Open Humans and the GET Conference." />
<meta property="og:description" content="For the past six years, this event has explored the frontiers of understanding about human biology. It serves as an annual forum to debate the technical, commercial, and societal impacts of advances in our ability to measure and understand people and their traits. The GET Conference is produced by PersonalGenomes.org, a 501(c)(3) charitable organization based in the United States whose mission is to make a wide spectrum of human biological information accessible and actionable to increase biological literacy and improve human health. We support groundbreaking initiatives including the Global Network of Personal Genome Projects, Open Humans and the GET Conference." />
<link rel="canonical" href="http://localhost:5000/pgpconference/get2014/labs.html" />
<meta property="og:url" content="http://localhost:5000/pgpconference/get2014/labs.html" />
<meta property="og:site_name" content="GET :: Genomes Environments Traits" />
<script type="application/ld+json">
{"description":"For the past six years, this event has explored the frontiers of understanding about human biology. It serves as an annual forum to debate the technical, commercial, and societal impacts of advances in our ability to measure and understand people and their traits. The GET Conference is produced by PersonalGenomes.org, a 501(c)(3) charitable organization based in the United States whose mission is to make a wide spectrum of human biological information accessible and actionable to increase biological literacy and improve human health. We support groundbreaking initiatives including the Global Network of Personal Genome Projects, Open Humans and the GET Conference.","author":{"@type":"Person","name":"GitHub User"},"@type":"WebPage","url":"http://localhost:5000/pgpconference/get2014/labs.html","headline":"About GET Labs","@context":"http://schema.org"}</script>
<!-- End Jekyll SEO tag -->
<link rel="stylesheet" href="/pgpconference/assets/css/style.css"><link type="application/atom+xml" rel="alternate" href="http://localhost:5000/pgpconference/feed.xml" title="GET :: Genomes Environments Traits" /><link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png">
  <link rel="icon" type="image/png" sizes="32x32" href="/favicon/favicon-32x32.png">
  <link rel="icon" type="image/png" sizes="16x16" href="/favicon/favicon-16x16.png">
  <link rel="manifest" href="/favicon/site.webmanifest">
  <link rel="mask-icon" href="/favicon/safari-pinned-tab.svg" color="#e35e31">
  <link rel="shortcut icon" href="/favicon.ico">
  <meta name="msapplication-TileColor" content="#b91d47">
  <meta name="msapplication-config" content="/favicon/browserconfig.xml">
  <meta name="theme-color" content="#ffffff">
</head>
<body class="sub labs labs2014"><div id="oho"><header class="site-header">
  <div class="wrapper nowrap"><div id="logo" class="site-logo"><a class="site-title" rel="author" href="http://localhost:5000/pgpconference/get2016" itemprop="headline"><img src="http://localhost:5000/pgpconference/get2010/images/GET-logo.gif" alt="Genomes Environments Traits"></a></div><nav class="site-nav top-nav nav2014">
        <input type="checkbox" id="nav-trigger" class="nav-trigger" />
        <label for="nav-trigger">
          <span class="menu-icon">
            <svg viewBox="0 0 18 15" width="18px" height="15px">
              <path d="M18,1.484c0,0.82-0.665,1.484-1.484,1.484H1.484C0.665,2.969,0,2.304,0,1.484l0,0C0,0.665,0.665,0,1.484,0 h15.032C17.335,0,18,0.665,18,1.484L18,1.484z M18,7.516C18,8.335,17.335,9,16.516,9H1.484C0.665,9,0,8.335,0,7.516l0,0 c0-0.82,0.665-1.484,1.484-1.484h15.032C17.335,6.031,18,6.696,18,7.516L18,7.516z M18,13.516C18,14.335,17.335,15,16.516,15H1.484 C0.665,15,0,14.335,0,13.516l0,0c0-0.82,0.665-1.483,1.484-1.483h15.032C17.335,12.031,18,12.695,18,13.516L18,13.516z" style="fill: white;" />
            </svg>
          </span>
        </label><ul class="trigger nav-links"><li><a href="/pgpconference/get2014/">2014 Home</a></li><li><a href="/pgpconference/get2014/speakers">Speakers</a></li><li><a href="/pgpconference/get2014/agenda">Agenda</a></li><li><a href="/pgpconference/get2014/sponsors">Sponsors</a></li><li><a href="/pgpconference/get2014/about">About</a></li><li><a href="/pgpconference/get2014/labs">Labs</a></li><li><a href="http://www.getconference.eventbrite.com/">Register</a></li></ul>
      </nav></div>
</header>
<main id="content" class="content" aria-label="Content"><h1 id="about-get-labs">About GET Labs</h1>

<p>Some of the most well-characterized humans on earth attend GET Labs. Many of these omic astronauts have already obtained personal genome sequencing, had their microbiomes characterized at various bodily habitats, and know something about the average length of their telomeres. Attendees often look forward to the opportunity to spit, swab, or be poked, measured, and surveyed for the good of science. The prospect of gaining access to these pioneers (and their data!) attracts an increasing number of research groups to join us for this event and organize phenotyping booths, specimen collection stations, sensor distribution and other activities.</p>

<p><strong>Attendance at GET Labs is by invitation only</strong>. This is a separate event from GET Conference, although both events are intimate and many people do participate in both. There are three ways to seek an invitation to this event:</p>

<ul>
  <li><strong>Research groups:</strong> <a href="https://docs.google.com/forms/d/1Uwqmmp0ARwTmuEmgnwxIWeEE10NUpf3d8py2z8QJYsU/viewform">Apply to attend</a>.</li>
  <li><strong>Harvard Personal Genome Project participants</strong>: Log-in and <a href="https://my.pgp-hms.org/third_party/18">apply to attend</a>.</li>
  <li><strong>Event sponsors</strong>: Email jason@personalgenomes.org</li>
</ul>

<p>While we do anticipate growing this event to support much broader audiences, currently we are focused on bringing together well-informed volunteers with extensive public data and researchers who wish to study them. At this point, our attendees are drawn from the Harvard Personal Genome Project (PGP). The Harvard PGP now has a cohort of more than 3000 eager, enrolled participants. These well-characterized individuals are enthusiastic and willing to contribute to the advancement of science in astounding ways. Many of them already have open-access whole genome data, health records, microbiome data, and cell lines.</p>

<div class="giant-text"><p class="margin-top-extra">Researchers will explore a wide range of human traits and their variability.</p></div>

<p><strong>GET LABS AGENDA FOR APRIL 29TH &amp; VENUE MAP:</strong> Download now as a <a href="/pgpconference/get2014/images/GETLABS_floorplan_agenda_04292014.pdf">PDF</a>.</p>

<p><a href="/pgpconference/get2014/images/GETLABS_floorplan_agenda_04292014.pdf"><img src="/pgpconference/get2014/images/getlabs_agenda_floorplan_500px.jpg" alt="GETLABS Floorplan Agenda 2015" /></a></p>

<ul class="labs-participants"><li class="lab-entry">
         <div class="lab-header">
           <p>
             American Gut Project<br />PI: Rob Knight, PhD. University of Colorado, Boulder</p>
         </div>
         <div class="lab-body">
           <div class="lab-about"><p>Ever since Antonie van Leeuwenhoek discovered a vast diversity of microbes between his own teeth in the 17th century using a primitive microscope, the microbes inhabiting our bodies have fascinated us. It is somewhat startling to realize that the human body has 10 times more non-human cells than 10 trillion human cells! It is only in the last few years that radical advances in DNA sequencing have allowed us to examine, for the first time, the full extent of microbial diversity on or in our bodies and to explore possible linkages between microbes and a wide range of health and disease states (including skin infections, neurological conditions, obesity, malnutrition, and inflammatory bowel disease). Principal Investigator: Rob Knight, PhD at the University of Colorado, Boulder.</p>
</div>
           <a class="lab-link"><img src="/pgpconference/get2013/images/microbiome_150.jpg" class="lab-img" /></a>
         </div>
       </li><li class="lab-entry">
         <div class="lab-header">
           <p>
             Armpit Microbes And You<br /></p>
         </div>
         <div class="lab-body">
           <div class="lab-about"><p>Although some may not admit it, we all have body odor. What you may not realize is that it isn’t you producing the odor, but it is the microscopic organisms living on you that produce your body odor. We are exploring microbial biodiversity on human skin (in the armpit) since the skin is the body’s first line of defense against pathogens and offers numerous opportunities to explore how a person’s microbial diversity can affect health and behavior. Our preliminary findings indicate that armpit product use significantly affects the microbes residing on the skin, but we have a lot to learn about how daily habits and underlying genetics influence the microbes living there. We will be sampling additional volunteers to identify correlations with health and daily habits, which will be greatly improved with datasets from PGP individuals.</p>

<p>Armpits are sexy and the smells produced by the microbes that live there may just help people choose their mate! We ask volunteers to go product-free for two days prior to sampling so we get a unique signature of the microbes living in the armpit without the inhibitory effects of products. Please stop by to learn more and to give us your sample! (Photo credit: Megan Ehlers)</p>
</div>
           <a class="lab-link"><img src="/pgpconference/get2014/images/armpit_microbes.jpg" class="lab-img" /></a>
         </div>
       </li><li class="lab-entry">
         <div class="lab-header">
           <p>
             Circles In Human Biology<br /></p>
         </div>
         <div class="lab-body">
           <div class="lab-about"><p>The age of personalized genomics offers an unprecedented chance to understand the biology of uniquely human traits. One interesting and understudied human-specific trait is the areola, a circle of specialized, pigmented skin surrounding the human nipple. Areolas have been proposed to play an intriguing role in human evolution; different features of the areola have been linked to sexual selection, nursing behavior, and infant weight gain. Developmentally, circular markings have a simple genetic architecture. This means that studying areolas can help us learn about the genetic underpinnings of distinctively human traits, while also laying important groundwork for improving breast and infant health.</p>

<p>The Tabin Laboratory at Harvard Medical School has engaged more than 200 participants from the Personal Genome Project to share data about their areolas as part of our effort to learn about the causes and consequences of diversity in this uniquely human trait. Using this data we have made several important discoveries: one, areolas are much more diverse than was previously recognized; and two, we now know that multiple, independent genetic pathways are responsible for the size and glandular structure of the areola.</p>

<p>We invite all PGP participants (both male and female) to join us in our effort to identify the genes responsible for variation in the appearance and glandular structure of the areola. Participation takes place in two parts. First, participants complete an online survey about health factors that could influence areola appearance, including weight, parity and breastfeeding history, hormonal status, etc. Second, participants perform a breast and areola self-characterization wherein they take measurements and photographs of their areolas (this is performed in the privacy of your own home). All measurements and images belong to participants and can be shared as you choose. In addition, all data will be uploaded to our secure server for analysis.</p>

<p>Please visit our 2014 GET Lab booth to pick up your areola self-characterization kit and to hear what we have been learning about the fascinating human areola. Not coming to GET Labs? We still want your help! If you are interested in participating in Circles, please read about the study and request a study kit by mail <a href="https://my.pgp-hms.org/third_party/12">here</a> (must be a Harvard PGP participant to access).</p>
</div>
           <a class="lab-link"><img src="/pgpconference/get2014/images/circles_in_human_evolution_150.jpg" class="lab-img" /></a>
         </div>
       </li><li class="lab-entry">
         <div class="lab-header">
           <p>
             Engaging With Personal Genomic Information<br />PI: Orit Shaer, PhD. Wellesley College</p>
         </div>
         <div class="lab-body">
           <div class="lab-about"><p>With your help, we aim to improve the design of websites and tools that make personal genomic information more accessible and understandable by the general public. We are academic researchers in the field of Human Computer Interaction that study opportunities and challenges for applying advanced HCI techniques in the area of personal genomics. We seek to learn more about the motivations, attitudes, and needs of users engaging with their personal genomic information. Thus, we look for volunteers who have already spent some time studying their own personal genomic data and want to tell us about their experiences.</p>

<p>We also plan to examine what human-computer interaction and visualization techniques and can help people to learn and generate new meaningful knowledge from their personal and family data. More specifically, we will explore how to design tools that enhance people’s understanding of their genomic data and help users to leverage the predictive mechanism of human health and disease, as well as how to design computational environments that facilitate novel models of collaboration between citizen and professional scientists, broadening the participation of the public in genomic investigations.(image credit: <a href="http://www.flickr.com/photos/irglover/7096373457/sizes/c/">flickr user irglover</a>)</p>
</div>
           <a class="lab-link"><img src="/pgpconference/assets/images/get2014/dna_irglover.jpg" class="lab-img" /></a>
         </div>
       </li><li class="lab-entry">
         <div class="lab-header">
           <p>
             Flu Near You<br /></p>
         </div>
         <div class="lab-body">
           <div class="lab-about"><p>New genetic variants of influenza and other pathogens are rapidly emerging. Since 2000, avian influenza viruses (H5N1, H7N7, and H7N3), human metapneumovirus, severe acute respiratory syndrome (SARS), coronavirus (CoV and human CoVs NL63 and HKU1, a new coronavirus (2012)), parvovirus types 4 and 5, mimivirus, H1N1 and initial cases of H3N2 and H1N2 in humans, have emerged and been identified. At the same time, global surveillance capacity has remained stagnant. Our current ability to estimate population-wide incidence of acute respiratory infection (ARI) and influenza-like illness (ILI) or measure the relative contributions of different pathogens to incidence are severely limited. Thus we at HealthMap, created Flu Near You (<a href="http://www.flunearyou.org">http://www.flunearyou.org</a>), an online participatory influenza surveillance tool for the USA and Canada.</p>

<p>Although the value of self-report systems such as Flu Near You have been demonstrated to engage individuals in the importance of influenza surveillance, enable them to access real-time information about what is directly happening in their community, and to generate detailed information about influenza risk for populations, none of the platforms have employed molecular techniques to rapidly verify the intelligence data garnered. Our goal is to enable Flu Near You participants who are willing to also provide samples for viral confirmation, to validate symptomatic reports as well as enable a better understanding of the representativeness of self-reported data and symptom profiles for different acute respiratory illnesses.</p>
</div>
           <a class="lab-link"><img src="/pgpconference/get2013/images/FluNearYou_150.jpg" class="lab-img" /></a>
         </div>
       </li><li class="lab-entry">
         <div class="lab-header">
           <p>
             Genome In A Bottle Consortium Giab<br /></p>
         </div>
         <div class="lab-body">
           <div class="lab-about"><p>As new high-throughput DNA sequencing methods are moving into clinical applications, understanding accuracy of genome sequencing instruments and methods will be critical. Numerous recent studies have demonstrated that analyzing the same sample on different sequencing technologies and bioinformatics analysis methods can yield very different results. There is a clear need for reference materials and quality metrics that may be used by device manufacturers and clinical laboratories to calibrate their machines and improve their operations.</p>

<p>With help from the Genome in a Bottle Consortium (www.genomeinabottle.org) and the FDA, the National Institute of Standards and Technology (NIST) is developing well-characterized whole human genomes as Reference Materials. Research and clinical laboratories will be able to purchase and use these Reference Materials to understand and improve the performance of their DNA sequencing operations and bioinformatics methods.</p>

<p>The NIST “Genome in a Bottle” Consortium has partnered with PersonalGenomes.org and the Harvard Personal Genome Project (PGP) to identify well-consented participants interesting in contributing to this important effort. Specimens donated by PGP volunteers are viewed as ideal candidates to serve as these new reference standards due to the depth and availability of public PGP datasets as well as the strength of the “open consent” process used in the Harvard PGP study. In particular, the Harvard PGP consent process addresses the risk of re-identification and enables the commercial use of donated tissues.</p>
</div>
           <a class="lab-link"><img src="/pgpconference/get2014/images/GIAB_reference_material.jpg" class="lab-img" /></a>
         </div>
       </li><li class="lab-entry">
         <div class="lab-header">
           <p>
             Genomics On Google Cloud Platform<br /></p>
         </div>
         <div class="lab-body">
           <div class="lab-about"><p>Generating research data is easier than ever before, but interpreting and analyzing it is still hard, and getting harder as the volume increases. This is especially true of genomics. Sequencing the whole genome of a single person produces more than 100 gigabytes of raw data, and a million genomes will add up to more than 100 petabytes.</p>

<p>This abundance of new information carries great potential for research and human health – and requires new standards, policies and technology. To contribute to the genomics community and help meet the data-intensive needs of the life sciences, Google recently introduced a simple web-based API to import, process, store, explore and collaborate with genomic data at “Google scale”.</p>

<p>With these first steps, it is our goal to make valuable data from projects like the Personal Genome Project widely accessible and useful. Together with the research community, we hope to improve access to larger sample sizes to distinguish between people who become sick and those who remain healthy, between patients who respond to treatment and those whose condition worsens, and between pathogens that cause outbreaks and those that are harmless. We aim to help researchers explore and analyze that information, even without access to a local supercomputer.</p>

</div>
           <a class="lab-link"><img src="/pgpconference/get2014/images/google_dna_150.jpg" class="lab-img" /></a>
         </div>
       </li><li class="lab-entry">
         <div class="lab-header">
           <p>
             Harvard Personal Genome Project<br /></p>
         </div>
         <div class="lab-body">
           <div class="lab-about"><p>The Personal Genome Project (PGP) is creating a freely available scientific resource that brings together genomic, environmental and human trait data. These data are donated by volunteers enrolled in a PGP study from our global network. Initiated by George Church at Harvard Medical School in 2005, the PGP has pioneered ethical, legal, and technical aspects related to the creation of public resources involving highly identifiable data like human genomes. The Harvard PGP began with a pilot study of ten fully identified individuals, known as the “PGP-10” and has incrementally scaled-up the cohort to more than 3000 enrolled volunteers.</p>

<p>We believe sharing is good for science and society. The PGP is dedicated to creating public resources that everyone can access. Anonymity is impossible to guarantee in a context like the PGP where public sharing of genetic data is an explicit goal. Therefore, the PGP collaborates with participants willing to waive expectations of privacy. This waiver is not for everyone, but the volunteers who join make a valuable and lasting contribution to science.</p>

<p>Our project aims to maintain lifelong relationships with participants. We want to collaborate on tracking health and other traits as they unfold over the course of a participant’s life. We also want to better understand the benefits and risks related to accessing and sharing extensive genomic and other personal data.</p>
</div>
           <a class="lab-link"><img src="/pgpconference/get2014/images/PGP_Logo_150.jpg" class="lab-img" /></a>
         </div>
       </li><li class="lab-entry">
         <div class="lab-header">
           <p>
             Health Data Exploration<br />Matthew Bietz &amp; Judith Gregory, University of California, Irvine<br />Kevin Patrick &amp; Jerry Sheehan, University of California San Diego</p>
         </div>
         <div class="lab-body">
           <div class="lab-about"><p>We are exploring the intersection of personal data and public science. There is considerable scientific value in making new forms of health data from wearable devices, smartphones, genetic testing, and other tools available to researchers in medicine, public health, and the social and behavioral sciences. Moreover, wider availability of these data might spark a new cadre of “citizen scientists” who can add to what traditional health researchers do. But for a variety of reasons it is not obvious how to make these data available for research. Issues to be addressed include whether there are unique scientific, methodological or ethical issues involved in such research; how to handle intellectual property of research findings; how and where these data intersect with other forms of medical and public health data; data quality; and privacy and confidentiality.</p>

<p>In order to better understand the experiences and concerns of individuals who share their data for scientific research, we will conduct a survey of GET Conference participants. We will also be conducting short interviews with participants during the event to gain a richer understanding of motivations for and concerns about participating in open science. Survey data and anonymized interview transcripts will be made publicly available. The results of this research will help shape research policy as well as the design of infrastructures for scientific data sharing.</p>
</div>
           <a class="lab-link"><img src="/pgpconference/get2014/images/hde_logo.jpg" class="lab-img" /></a>
         </div>
       </li><li class="lab-entry">
         <div class="lab-header">
           <p>
             How Does Genomic Information Affect Mesearch<br />Andrea Farkas Patenaude, Ph.D., Dana Farber Cancer Institute</p>
         </div>
         <div class="lab-body">
           <div class="lab-about"><p>As a participant in the Personal Genome Project, you are a pioneer. The story of how you and others came to participate, your experiences as a participant and the ways participation has or could change things in your life are of great interest to experts in ethics and psychology. We are interested in how individuals utilize information about genes affecting their families. We would like to interview a wide range of participants about their PGP experiences. Interviews will be confidential, take 40-60 minutes, and occur at a time you choose. Data from the interviews and a larger survey we will develop based on the interviews will help us and others in consideration of how genomic research might be conducted in the future and help us to know how best to involve families in genomic research. Please stop by and talk with us at the GET conference. If you won’t be at the GET Conference, but would consider being interviewed, please send your name, email address and phone number to the project PI, Dr. Andrea Patenaude, at andrea_patenaude@dfci.harvard.edu and we will call you. We are happy to answer questions and, if you are willing, we will add you to the list of participants who will be contacted for interviews over the next few months. We have so much to learn from you!</p>
</div>
           <a class="lab-link"><img src="/pgpconference/get2014/images/Andrea-Patenaude.jpg" class="lab-img" /></a>
         </div>
       </li><li class="lab-entry">
         <div class="lab-header">
           <p>
             Limb Lab<br />Abby Wark, PhD, Harvard Medical School</p>
         </div>
         <div class="lab-body">
           <div class="lab-about"><p>Our hands shape our interactions with the world. They let us throw, touch, and manipulate objects in intricate ways. In cultures all around the world, people use their hands to greet each other and to communicate without speaking. Research has begun to reveal how animal limbs evolve and develop, but human hands are unique and remain mysterious. Compared to our nearest primate relatives, we have short fingers, long thumbs and special grip capabilities. We want to understand how these unique human traits are built. What are the genes that enabled the evolution of human limbs? What causes variation in hand bone growth and development? How do we come to have hands that are definitively human and yet distinct in each of us?</p>

<p>This year at GET, we are offering all participants a chance to enroll in the Limb Lab, a new research study of diversity in human hand and limb characteristics. We will perform on-site analyses of hands, arms and legs using non-invasive technologies, including photography, scanning/photocopying, and bone length measurements. Please stop by and let your hands do the talking as you help us unlock the mysteries of how human bodies are built.</p>

<p>Not coming to GET? If you are interested in participating in the Limb Lab after GET2014, please email: awark@genetics.med.harvard.edu (Note: all participation must take place in Boston)</p>
</div>
           <a class="lab-link"><img src="/pgpconference/get2014/images/LimbLabLogo_150x203.jpg" class="lab-img" /></a>
         </div>
       </li><li class="lab-entry">
         <div class="lab-header">
           <p>
             Of Mites And Men<br />Daniel Fergus, North Carolina Museum of Natural Sciences</p>
         </div>
         <div class="lab-body">
           <div class="lab-about"><p>We live in the age of DNA sequencing, and yet there is an animal for which we have almost no DNA sequence even though it seems to be living closely with almost every one of us. This animal is literally right under our noses, as well as pretty much everywhere else on our faces. Mites of the genus Demodex make their homes in the pores and hair follicles of almost every adult, coming out at night to move around and breed. They have been long known among dermatologists and veterinarians for their association with rosacea and mange. However, their small size and elusive lifestyle have made it difficult to get a comprehensive understanding of our symbiotic mites. We aim to change this. We are collecting Demodex mites from both humans and non-humans, using DNA sequence data to look at the evolutionary relationships among the mites from different host and different parts of the body, and examining the microbial symbionts of the mites and their potential relationship to associated skin conditions. We are particularly interested in sampling people from other countries and people with conditions like blepheritis and rosacea. Sampling PGP participants will be a great opportunity to examine Demodex populations in the context of all the valuable information that participants have made available. (Photo credit: Daniel Fergus)</p>
</div>
           <a class="lab-link"><img src="/pgpconference/get2014/images/demodex_mite.jpg" class="lab-img" /></a>
         </div>
       </li><li class="lab-entry">
         <div class="lab-header">
           <p>
             Open Humans<br />Madeleine Ball and Jason Bobe, PersonalGenomes.org</p>
         </div>
         <div class="lab-body">
           <div class="lab-about"><p>Open Humans is a new project by PersonalGenomes.org to transform data sharing in human health and biology. Drawing on our work with the Personal Genome Project and GET Labs, Open Humans aims to facilitate two types of sharing: researchers sharing data with participants, and participants aggregating and sharing data publicly. We think this idea has the potential to make a huge impact. Involution Studios is helping us gather vital feedback from both researchers and participants from this amazing community. Please help us design this exciting project by joining us for a 30 minute interview! Learn more about Open Humans at our new website: <a href="http://www.openhumans.org">www.openhumans.org</a></p>

</div>
           <a class="lab-link"><img src="/pgpconference/get2014/images/openhumans_150.jpg" class="lab-img" /></a>
         </div>
       </li><li class="lab-entry">
         <div class="lab-header">
           <p>
             Open Source Physiological Monitors<br />Charles Fracchia, MIT Center for Bits and Atoms</p>
         </div>
         <div class="lab-body">
           <div class="lab-about"><p>Technologies for rudimentary physiological sensing are becoming cheaper and easier to implement, leading to a proliferation of start-ups in the quantified self and body sensor spaces. To accelerate progress, we are developing well-documented open-source hardware and software platforms to enable the integration and customization of multiple sensing modalities for long-term monitoring. We will present a functional open source EKG which can be adapted for EEG or EMG measurements, as well as early prototypes of a BlueTooth compatible Bio-Glasses (<a href="http://bioglasses.org/">www.bioglasses.org</a>) platform that integrates three or more physiological sensors, as well as motion monitoring, into a pair of glasses. These systems are fully open and are designed allow creative modification by non-engineers.</p>
</div>
           <a class="lab-link"><img src="/pgpconference/get2013/images/spike.jpg" class="lab-img" /></a>
         </div>
       </li><li class="lab-entry">
         <div class="lab-header">
           <p>
             Perceptions Of Ancestry<br />Principal Investigator: Jennifer K. Wagner JD PhD, University of Pennsylvania</p>
         </div>
         <div class="lab-body">
           <div class="lab-about"><p>Siblings who share the same biological parents (and, by extension, the same ancestors) may self-identify differently when asked about race; may look quite different in terms of physical appearance (including skin color, eye color, hair color and texture, and facial features); may be perceived by others as having different ancestries; and may even have different proportional genomic ancestries. The relationships between race, identity, appearance, and genealogical, genomic, and perceived ancestry are not well understood. How individuals identify and are identified by others is shaped to an unknown degree by our expectations about how an individual of a particular ancestry “should” look. The level of discordance between an individual’s appearance, perceived ancestry, and proportional genomic ancestry may affect the formation and stability of an individual’s racial identity, the frequency and severity of discrimination the individual experiences, and even the individual’s own prejudices. By better understanding perceived ancestry, we may be able to develop effective tools to mitigate implicit prejudices in medical and legal contexts.</p>

<p>This phase of the “Multidisciplinary Study of Race, Ancestry, Appearance, Discrimination &amp; Prejudice,” funded by the National Human Genome Research Institute’s Ethical, Legal, and Social Implications Program, explores how individuals conceptualize and visualize ancestry and variation in human faces. Using Survey Monkey instruments, participants are first asked background questions about themselves and then shown a set of 3D, animated facial images with corresponding questions. The facial images used as stimuli in the perception tasks are from individuals who agreed to participate in research related to perceiving ancestry and genes in facial features and for whom substantial information is available to compare with observers’ perceptions of ancestry.</p>

<p>This project dovetails a research project led by Dr. Mark Shriver at Penn State University. Participants of both studies may have additional ancestry perception task opportunities at GET Labs.</p>

</div>
           <a class="lab-link"><img src="/pgpconference/get2014/images/jen.jpg" class="lab-img" /></a>
         </div>
       </li><li class="lab-entry">
         <div class="lab-header">
           <p>
             Pgp Stem Cell Lines Made Using A Fully Automated System<br />The New York Stem Cell Foundation Research Institute, New York, NY 10023</p>
         </div>
         <div class="lab-body">
           <div class="lab-about"><p><a href="http://en.wikipedia.org/wiki/Induced_pluripotent_stem_cell">Induced pluripotent stem (iPS) cells</a> are derived from adult skin cells and can give rise to any cell type within the human body. This makes them a promising research focus for improving human health. PGP participants are a great resource for scientists, especially those studying stem cells, because their genome and other biological data can be shared publicly. The generation of iPS cell lines from PGP participants would have tremendous utility for the scientific community in both basic and clinical research by providing relevant information, inaccessible from other cohorts, that is related to the donor and can be linked to the iPS cell line. To ensure the longevity, quality, and consistency of the PGP iPS cell lines, they must be generated using high-quality, consistent procedures. The New York Stem Cell Foundation (NYSCF) Research Institute has developed robotic methods to create standardized cells, which help scientists understand when cell features are due to genetic or epigenetic differences (and not to the different techniques used to create the cell lines). By working together, NYSCF and PGP will create standard iPS cell lines whose data can be published publicly – a great resource for the scientific community.</p>

<p>The NYSCF robotic platform, known as the NYSCF Global Stem Cell Array, is a new technology platform for the derivation and manipulation of stem cell lines as well as the differentiation of adult cell types in a high-throughput, parallel process using automation. Standardization and scale-up capabilities achieved through this automated process are critical in efforts to reduce methodological variability to uncover true biology by improving signal to noise ratios in phenotype studies. The automated system has the capacity to generate hundreds of stem cell lines per month allowing large-scale investigation into disease pathologies and drug toxicity and efficacy studies. The platform will generate panels of stem cell lines from thousands of genetically diverse individuals representing both diseased patients and controls. If you are interested in participating, please visit the PGP (http://www.personalgenomes.org/) or NYSCF <a href="http://www.nyscf.org/get-involved/volunteering">http://www.nyscf.org/get-involved/volunteering</a> website. Make sure to mention that you are a PGP participant. (Image credit: <a href="http://www.flickr.com/photos/pennstatelive/8972110324/">PennStateLive</a> <a href="http://creativecommons.org/licenses/by-nc-nd/2.0/">CC-BY-NC-ND</a>)</p>

</div>
           <a class="lab-link"><img src="/pgpconference/get2014/images/adult_stem_cells_neurons.jpg" class="lab-img" /></a>
         </div>
       </li><li class="lab-entry">
         <div class="lab-header">
           <p>
             Screening For Sleep And Circadian Rhythm Disorders<br />Brian Cade, Brigham and Women's Hospital</p>
         </div>
         <div class="lab-body">
           <div class="lab-about"><p>Sleep occupies a substantial portion of our lives and is increasingly being recognized for its role in cardiovascular disease, cancer, memory, safety and performance, and all-cause mortality. Yet we have been sleeping in potentially maladaptive situations since the invention of the light bulb, in a society increasingly reliant on shift-work and 24-hour connectivity.</p>

<p>Despite connections to a broad range of physiology, the genetics of sleep remain poorly understood. Questionnaire screening for sleep and circadian rhythm disorders can establish a baseline to quantify relationships with other measured non-sleep traits, generate pilot data and methods for future well-powered studies, and provide state-of-the-art data for areas with minimal genetic study (e.g. owl-lark preference). Our multi-disciplinary team has expertise in sleep, circadian rhythms, and quantitative genomics. We are currently working on multiple projects, including the largest genetic meta-analysis study of obstructive sleep apnea to date. (Image credit: <a href="http://www.flickr.com/photos/thejesse/413461932/">flickr user thejesse</a>, <a href="http://creativecommons.org/licenses/by-sa/2.0/deed.en">CC-BY-SA 2.0</a>)</p>
</div>
           <a class="lab-link"><img src="/pgpconference/get2014/images/sleep_thejesse_150.jpg" class="lab-img" /></a>
         </div>
       </li></ul>

<div class="labs-footer2014">
  
<p><strong>Suggest a GET Lab:</strong> If you want to recommend a researcher or research project, <a href="http://www.getconference.org/get2014/contact.html">we welcome your suggestions</a>. Thank you!</p>

<p><strong>Fellows program:</strong> We provide competitive support to researchers who wish to bring their research to GET Labs. <a href="http://www.getconference.org/get2014/fellows.html">Learn more</a>. Update: Thanks to all the applicants, all fellows have been selected and notified for the 2014 program.</p>

</div>

    </main></div><footer class="site-footer h-card">
  <data class="u-url" href="/pgpconference/"></data>

  <div class="container">
    <div class="row">
      <div class="col-xs-12">
        Copyright © 2019 <a href="http://localhost:5000/pgpconference">GET :: Genomes Environments Traits</a>
      </div>
    </div> <!-- contact .row -->
  </div>

</footer>
</div>
  </body>

</html>
